### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by Robinnnnn**

### CODE REVIEW

File: src/components/GlobalNavbar/LoggedInNav.tsx [hunk 1]
```diff
@@ -28,6 +28,11 @@ function LoggedInNav() {
   }, [userAddress]);
 
   const handleGalleryRedirect = useCallback(() => {
+    const authenticatedUserIsOnTheirOwnPage =
+      window.location.pathname.slice(1) === user.username;
+    if (authenticatedUserIsOnTheirOwnPage) {
+      return;
+    }
```
Comment (Robinnnnn) [inline src/components/GlobalNavbar/LoggedInNav.tsx:35]: prevent unnecessary redirect if user is already on their own page

File: src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx [hunk 2]
```diff
@@ -39,6 +40,14 @@ function CollectionEditor() {
 
   const unassignedNfts = useUnassignedNfts({ skipCache: false });
 
+  const { collections } = useAuthenticatedGallery();
+  const nftsWithinCollectionBeingEdited = useMemo(() => {
+    const collectionBeingEdited = collections.find(
+      (coll) => coll.id === collectionIdBeingEdited
+    );
+    return collectionBeingEdited?.nfts ?? [];
+  }, [collectionIdBeingEdited, collections]);
```
Comment (Robinnnnn) [inline src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx:49]: this is mostly working, still some weird bug with the context

File: src/hooks/api/collections/useCreateCollection.ts [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
         CreateCollectionResponse,
         CreateCollectionRequest
       >('/collections/create', 'create collection', {
         gallery_id: galleryId,
+        name: title,
+        collectors_note: description,
         nfts: nftIds,
       });
 
+      mutate(getGalleriesCacheKey({ userId: authenticatedUser.id }));
```
Comment (Robinnnnn) [inline src/hooks/api/collections/useCreateCollection.ts:29]: after creating collection, refresh get gallery endpoint so user will see latest

File: src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx (hunk 2 above)
Comment (kaitoo1) [reply src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx:49]: What's the bug? Should we leave a todo to remember

File: src/flows/shared/steps/OrganizeGallery/Header.tsx [hunk 4]
```diff
@@ -1,15 +1,16 @@
 import { useCallback } from 'react';
 import styled from 'styled-components';
 import Button from 'components/core/Button/Button';
-import { Heading } from 'components/core/Text/Text';
-import TextButton from 'components/core/Button/TextButton';
+import { Caption, Heading } from 'components/core/Text/Text';
+// import TextButton from 'components/core/Button/TextButton';
```
Comment (kaitoo1) [inline src/flows/shared/steps/OrganizeGallery/Header.tsx:5]: Delete

File: src/flows/shared/steps/OrganizeGallery/Header.tsx [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
 import Spacer from 'components/core/Spacer/Spacer';
 import { withWizard, WizardComponentProps } from 'react-albus';
+import colors from 'components/core/colors';
 
 function Header({ wizard: { push } }: WizardComponentProps) {
-  const handleGalleryPreview = useCallback(() => {
-    alert('TODO - go to gallery preview');
-  }, []);
+  // const handleGalleryPreview = useCallback(() => {
```
Comment (kaitoo1) [inline src/flows/shared/steps/OrganizeGallery/Header.tsx:11]: Delete?

File: src/scenes/Nuke/Nuke.tsx [hunk 6]
```diff
@@ -1,20 +1,33 @@
...
+      // this will only work on dev!
+      await fetcher('/nuke', 'nuke database');
+    } catch (e) {
+      // error silently
+    }
+  }, [fetcher]);
+
+  useEffect(() => {
+    logOut();
+    nukeDb();
```
Comment (kaitoo1) [inline src/scenes/Nuke/Nuke.tsx:29]: Maybe we can check the env and only call this if dev, so we don't make the request at all on prod

**Review by kaitoo1 — APPROVED** [review]

File: src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx (hunk 2 above)
Comment (Robinnnnn) [reply src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx:49]: fixed!

**Pull request merged**